src/sqlancer/mysql/ast/MySQLCastOperation.java

```diff
@@ -6,7 +6,7 @@ public class MySQLCastOperation implements MySQLExpression {
     private final CastType type;
 
     public enum CastType {
-        SIGNED, UNSIGNED;
+        SIGNED, UNSIGNED , INT, BOOLEAN ,TEXT;
```

Comment:
Please keep the PR focused on a single change. We can review and merge changes more quickly this way.